kata-deploy/README.md

```diff
@@ -93,7 +93,7 @@ is defined through a custom resource definition. For Kubernetes 1.13:
   $ kubectl apply -f https://raw.githubusercontent.com/kata-containers/packaging/master/kata-deploy/k8s-1.13/runtimeclass-crd.yaml
 ```
 
-In order to use a workload Kata with QEMU, first add a `RuntimeClass` as:
+To use a workload with Kata-QEMU, first add a `RuntimeClass` as:
```

Comment:
I may have missed previous discussion but I'm afraid I'm not a fan of this (new) term. I think "Kata for QEMU" is the clearest way to refer to this, or alternatively you could refer to "the `kata-qemu` runtime class".

Same comment for the text below.

Reply:
Sure @jodh-intel . Updated.